app/javascript/dashboard/components/widgets/TableFooter.vue

```diff
@@ -1,171 +1,168 @@
 <template>
   <footer
     v-if="isFooterVisible"
-    class="bg-white dark:bg-slate-800 h-12 border-t border-solid border-slate-75 dark:border-slate-700/50 flex items-center justify-between px-6"
+    class="bg-white dark:bg-slate-900 h-12 flex items-center justify-between px-6"
   >
-    <div class="left-aligned-wrap">
-      <div class="text-xs text-slate-600 dark:text-slate-200">
-        <strong>{{ firstIndex }}</strong>
-        - <strong>{{ lastIndex }}</strong> of
-        <strong>{{ totalCount }}</strong> items
-      </div>
-    </div>
-    <div class="right-aligned-wrap">
+    <span class="text-sm text-slate-700 dark:text-slate-200 font-medium">
```

Comment:
In case there are no extra pages, and all the items are shown, we can say "Showing all 26 items"

CC: @absurdiya wdyt?

Reply:
> In case there are no extra pages, and all the items are shown, we can say "Showing all 26 items"
> 
> CC: @absurdiya wdyt?

Sounds good